src/ott/datasets.py

```diff
@@ -105,6 +110,168 @@ def _create_sample_generators(self) -> Iterator[jnp.array]:
       yield samples
 
 
+@dataclasses.dataclass
+class SklearnDistribution:
```

Comment:
Please add this and all the newly created functions to the docs in `docs/datasets.rst`; I think you will need to create this file - please have a look at `docs/utils.rst` how it's done.